wgpu/gui/offscreen.py

```diff
@@ -3,9 +3,10 @@
 import numpy as np
 
 from ._offscreen import WgpuOffscreenCanvas
+from .base import WgpuAutoGui
 
 
-class WgpuManualOffscreenCanvas(WgpuOffscreenCanvas):
+class WgpuManualOffscreenCanvas(WgpuAutoGui, WgpuOffscreenCanvas):
```

Comment:
I don't think this one has `self._event_handlers` now

Reply:
Good catch.